django/db/backends/postgresql/introspection.py

```diff
@@ -115,31 +115,18 @@ def get_relations(self, cursor, table_name):
         Return a dictionary of {field_name: (field_name_other_table, other_table)}
         representing all relationships to the given table.
         """
+        return {row[0]: (row[2], row[1]) for row in self.get_key_columns(cursor, table_name)}
```

Comment:
It would be a bit more clear like:

```
{field_name: (field_name_other_table, other_table) 
 for field_name, other_table, field_name_other_table in self.get_key_columns(cursor, table_name)}
```

Reply:
It's already described in the docstring and it's quite common pattern in our introspection methods, so I think it's fine to leave it as it is.